### ISSUE TITLE
Feedback mini_ex3 by Rolf Holm

### ISSUE THREAD

**Issue by OorthianEmissary** [duendue/Aesthetic-Programming-2017#11]
This throbber is a nostalgic take back to The Portal games ending theme, with the logo and animated throbber.
The added ending song and cake makes for a nice touch that completes what the program is trying to show.
This throbber is a perfect reference in itself.
I having nothing to add.

As a contribution to digital culture it's more of a remake of an old logo, now with added throbber animation for the Aperature Science logo circle.

In conclusion; Great reference, great program, flawless execution.
